### PR TITLE
Fix vaults token key name and adding tests

### PR DESCRIPTION
**Pull request by WHYTEWYLL**

### CODE REVIEW

File: macros/api_udf/request_repo_data.sql [hunk 1]
```diff
@@ -17,7 +17,7 @@ CREATE TABLE IF NOT EXISTS {{ target.database }}.bronze_api.github_repo_data(
 
 {% set event_table %}
 
-CREATE TABLE IF NOT EXISTS {{ target.database }}.bronze_api.log_messages (
+CREATE  OR REPLACE TABLE {{ target.database }}.bronze_api.log_messages (
```
Comment (eric-laurello) [inline macros/api_udf/request_repo_data.sql:20]: do you want to drop all the old data on each run?

File: tests/github_data_null.sql [hunk 2]
```diff
@@ -0,0 +1,22 @@
+WITH base AS (
```
Comment (eric-laurello) [inline tests/github_data_null.sql:1]: needs to use the __ to denote a schema name for where the test results will land

File: tests/github_data_null.sql [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    SELECT
+        SPLIT_PART(MESSAGE, ',',2) AS status_code,
+        count(*) as counter
+    FROM
+        crosschain.bronze_api.log_messages
```
Comment (eric-laurello) [inline tests/github_data_null.sql:6]: should use a ref

File: tests/github_data_null.sql [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    WHERE
+        timestamp BETWEEN DATEADD(HOUR, -1, CURRENT_TIMESTAMP())
```
Comment (eric-laurello) [inline tests/github_data_null.sql:8]: not sure how frequently the test or model runs but you may need to lookback further. Also we should be using SYSDATE() vs current_date.  sysdate guarantees utc time and current date is based on a snowflake setting

File: macros/api_udf/request_repo_data.sql (hunk 1 above)
Comment (WHYTEWYLL) [reply macros/api_udf/request_repo_data.sql:20]: yeah

File: tests/github_data_null.sql (hunk 4 above)
Comment (WHYTEWYLL) [reply tests/github_data_null.sql:8]: One after the run, that why Im deleting the table when the procedure runs

**Review by eric-laurello — APPROVED** [review]

**Pull request merged**